### PR TITLE
Log cleanup

### PR DESCRIPTION
**Pull request by ansnunez**
## Description

Some log improvements:
1. Reduced usage of `LOG_EPOCH`
2. Shortened some of the more frequent messages, such as for p2p and gossip
3. Reduced payload lengths
4. More consistent display of committee/shard/peer lists
5. Overall, tried to make logs more concise and able to show in single line
6. Also added temp var before calling `LIMIT_RIGHT` in logger, to avoid repeated string creation inside the macro

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: src/libDirectoryService/DSBlockPreProcessing.cpp [hunk 1]
```diff
@@ -272,17 +272,19 @@ void DirectoryService::InjectPoWForDSNode(VectorOfPoWSoln& sortedPoWSolns,
...
       m_allPoWConns.emplace(m_mediator.m_DSCommittee->at(
           m_mediator.m_DSCommittee->size() - 1 - i));
-      LOG_GENERAL(INFO, m_mediator.m_DSCommittee
-                            ->at(m_mediator.m_DSCommittee->size() - 1 - i)
-                            .second);
+      LOG_GENERAL(INFO,
+                  "Injecting into PoW connections "
+                      << m_mediator.m_DSCommittee
+                             ->at(m_mediator.m_DSCommittee->size() - 1 - i)
+                             .second);
```
Comment (ShengguangXiao) [inline src/libDirectoryService/DSBlockPreProcessing.cpp]: Can assign "m_mediator.m_DSCommittee->size() - 1 - i" to a temp variable?

File: src/libDirectoryService/PoWProcessing.cpp [hunk 2]
```diff
@@ -316,19 +312,17 @@ bool DirectoryService::ProcessPoWSubmissionFromPacket(
...
     // Accept slightly late entries as we need to multicast the DSBLOCK to
     // everyone if ((m_state != POW_SUBMISSION) && (m_state !=
     // DSBLOCK_CONSENSUS_PREP))
     if (!CheckState(VERIFYPOW)) {
-      LOG_EPOCH(INFO, m_mediator.m_currentEpochNum,
-                "Too late - current state is " << m_state);
+      LOG_GENERAL(INFO, "Too late. State = " << GetStateString());
     } else {
-      LOG_EPOCH(INFO, m_mediator.m_currentEpochNum, "POW verification passed");
+      LOG_GENERAL(INFO, "Verified OK");
```
Comment (ShengguangXiao) [inline src/libDirectoryService/PoWProcessing.cpp]: Log as "POW verified OK" will be more clear

Comment (ansnunez) [reply src/libDirectoryService/PoWProcessing.cpp]: Won't be necessary, since you can already see from the function name that it is for checking PoW submission:

```
[INFO][15306][19-01-24T14:17:43.918][oWProcessing.cpp:325][ProcessPoWSubmission] Verified OK
```

I tried to reduce the logs as much as possible, avoid being too wordy and have too much info.

Comment (ansnunez) [reply src/libDirectoryService/PoWProcessing.cpp]: BTW here is a fun fact, if we add something as small as "PoW ":

"PoW " = 4 bytes
4 bytes x 1800 submissions/epoch x 12 epochs/day x 365 days/year = 30MB per year!
